### ISSUE TITLE
empty page after upgrade

### ISSUE THREAD

**Issue by tps800** [owncloud/core#12095]
ownCloud shows only an empty page after last upgrade
![nc158v3_108_https___lasair bfs de_owncloud_ - chromium](https://cloud.githubusercontent.com/assets/702156/4989655/ca8f4c0c-697b-11e4-8342-3e9dd812b631.jpg)

**Comment by stv0g** [comment]
I'm getting the same error with 8.0 RC from the latest daily archive.
But only after i've installed the calendar and contact apps.

**Comment by jnweiger** [comment]
@stv0g Thanks for testing RC1. Did you install from tar.gz or deb or rpm package?

**Comment by stv0g** [comment]
I downloaded the nightly tar.gz archive from yesterday.

**Comment by VicDeo** [comment]
@stv0g where did you take calendar and contact from?

**Comment by stv0g** [comment]
Calendar: rework branch of git
Contacts: master branch of git

**Comment by onny** [comment]
I encounter this issue with the latest stable owncloud 8 release

**Comment by einmalacht** [comment]
Same problem here when i upgrade to owncloud 8

**Comment by karlitschek** [comment]
@einmalacht Have you manually installed  some apps?
Can you post the full bug report using the issue template please? 
https://github.com/owncloud/core/blob/master/CONTRIBUTING.md

**Comment by oksiquatzel** [comment]
Hi 
as i upgraded from 7.0.4 to 8.0.0 with debian aptitude and opensuse repo, i had the same error. 
i found out that the 
files_mv app 
requies the chosen/chosen.jquery.min
after removing the app my owncloud instance worked.

**Comment by dirtyharry123** [comment]
I can confirm this error appeared when updating to 8.0.0 using opensuse repo on debian.
The solution mentioned by @oksiquatzel works

**Comment by HakanKoseoglu** [comment]
Replicated on Ubuntu 14.04 server. I don't think I have the files_mv app and not sure how to figure out from where this comes from:

{"reqId":"973a8c9b1583d1132d512e3707ee1412","remoteAddr":"192.168.1.65","app":"PHP","message":"js file not found: script:js\/chosen\/chosen.jquery.min serverroot:\/var\/www\/owncloud at \/var\/www\/owncloud\/lib\/private\/template\/resourcelocator.php#46","level":3,"time":"2015-02-11T09:10:08+00:00"}

At least I know resourcelocator.php is owned by the server:
root@jupiter:/var/www/owncloud/lib/private/template# dpkg -S $PWD/resourcelocator.php 
owncloud-server: /var/www/owncloud/lib/private/template/resourcelocator.php

**Comment by sjstoelting** [comment]
I can confirm the empty page after update from 7.0.4 to 8.0.0. The update was done by manually upload.

After I removed the calendar and the contacts apps, my installation worked again.
I have used the original calendar and contacts apps

**Comment by NHellFire** [comment]
Could owncloud/3rdparty#138 be reverted? It breaks some addons, such as [Files Move](https://apps.owncloud.com/content/show.php?content=150271) and apparently also (I don't use this one) [Shorty](https://apps.owncloud.com/content/show.php/Shorty?content=150401)

**Comment by DeepDiver1975** [comment]
> Could owncloud/3rdparty#138 be reverted? It breaks some addons, such as Files Move and apparently also (I don't use this one) Shorty

No - sorry - chosen is no longer needed in owncloud core and apps should never have had a dependency on it. The apps need to fix this themselves.



> In owncloud/3rdparty, you need to download the patch that removes it, then revert it,

That's the wrong approach - the apps need to fix this by themselves. THX

**Comment by DeepDiver1975** [comment]
I'm closing this issue because the error is related to apps which have not yet been migrated to owncloud 8.

**Comment by cmorty** [comment]
@DeepDiver1975 I can confirm this issue with the stock calender after using the updater App. When disabling the calender everything is fine. Please reopen, this is really a PITA.
